dotCMS/src/main/java/com/dotcms/rest/api/v1/secret/DeleteSecretForm.java

```diff
@@ -0,0 +1,41 @@
+package com.dotcms.rest.api.v1.secret;
+
+import com.dotcms.repackage.javax.validation.constraints.NotNull;
+import com.dotcms.rest.api.Validated;
+import java.util.Set;
+
+public class DeleteSecretForm extends Validated {
+
+    @NotNull
+    private String serviceKey;
+
+    @NotNull
+    private String siteId;
+
+    @NotNull
+    private Set<String> params;
+
+    public String getServiceKey() {
+        return serviceKey;
+    }
+
+    public void setServiceKey(String serviceKey) {
+        this.serviceKey = serviceKey;
+    }
+
+    public String getSiteId() {
+        return siteId;
+    }
+
+    public void setSiteId(String siteId) {
+        this.siteId = siteId;
+    }
+
+    public Set<String> getParams() {
+        return params;
+    }
+
+    public void setParams(Set<String> params) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'params' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4623512)